cilium-cli/cli/connectivity.go

```diff
@@ -95,12 +95,12 @@ func RunE(hooks api.Hooks) func(cmd *cobra.Command, args []string) error {
 			return nil
 		}
 
-		logger := check.NewConcurrentLogger(params.Writer, params.TestConcurrency)
-		owners, err := codeowners.ParseFile(strings.NewReader(assets.CodeOwnersRaw))
+		owners, err := owners_util.Load(params.CodeOwners)
 		if err != nil {
-			return fmt.Errorf("🐛 Failed to parse CODEOWNERS. Developer BUG? %w", err)
+			return fmt.Errorf("❗ Failed to load code owners: %w", err)
```

Comment:
@joestringer Does this need to be a hard failure? It looks like it is now no longer possible to run `cilium connectivity test` from outside the Cilium directory (without passing extra parameters), as it immediately fails with `❗ Failed to load code owners: while loading: could not find CODEOWNERS file at any of the standard locations`.

Reply:
Related to that, the `cilium connectivity perf` does not take the `--code-owners` parameter, so there's currently no workaround besides running it inside the Cilium tree.

Reply:
We can relax this by putting this under the `--log-code-owners` option. PR:

https://github.com/cilium/cilium/pull/38133